lib/scan/detect_values.rb

```diff
@@ -23,6 +23,22 @@ def self.set_additional_default_values
       return config
     end
 
+    def self.filter_simulators(simulators, deployment_target)
+      # Filter out any simulators that are not the same major and minor version of our deployment target
+      if deployment_target.to_s.length > 0
```

Comment:
This looks like a lot of code for what it does. Basically we're only comparing that the major version matches?